collections/sea-ice-index.yaml

```diff
@@ -44,7 +44,7 @@ Resources:
     Role: processor
     Type: byoc-10549890-13bb-4ec5-8ae0-df387bd0b785
     CollectionId: 10549890-13bb-4ec5-8ae0-df387bd0b785
-    Primary: true  
+    Primary: true     
```

Comment:
You added again some empty lines. Please remove those. Afterwards you can merge this :)

Reply:
Removed. Need approval for merging :)